lang/rust/rust.talon

```diff
@@ -0,0 +1,64 @@
+mode: command
+and mode: user.rust
+mode: command
+and mode: user.auto_lang
+and code.language: rust
+-
+tag(): user.code_imperative
+tag(): user.code_object_oriented
+
+tag(): user.code_comment_line
+tag(): user.code_comment_documentation
+tag(): user.code_data_bool
+tag(): user.code_data_null
+tag(): user.code_functions
+tag(): user.code_functions_gui
+tag(): user.code_libraries
+tag(): user.code_libraries_gui
+tag(): user.code_operators_array
+tag(): user.code_operators_assignment
+tag(): user.code_operators_bitwise
+tag(): user.code_operators_math
+
+settings():
+    user.code_private_function_formatter = "SNAKE_CASE"
+    user.code_protected_function_formatter = "SNAKE_CASE"
+    user.code_public_function_formatter = "SNAKE_CASE"
+    user.code_private_variable_formatter = "SNAKE_CASE"
+    user.code_public_variable_formatter = "SNAKE_CASE"
+
+unsafe:
+    "unsafe {\n\n}"
```

Comment:
What if I want to write `unsafe fn`? Seems like it'd be reasonable to just insert "unsafe " so I could say `state unsafe` then either `block` or `funky foo`.